docs/bosh-troubleshooting-gcp.adoc

```diff
@@ -0,0 +1,59 @@
+= BOSH Troubleshooting
```

Comment:
The idea was to move it into the BOSH Troubleshooting lab which comes later in the course.